src/Entity/OgRole.php

```diff
@@ -220,6 +220,13 @@ public static function loadByGroupAndName(EntityInterface $group, $name) {
   /**
    * {@inheritdoc}
    */
+  public static function loadByGroupTypeGroupBundleAndName($entity_type_id, $bundle, $role_name) {
```

Comment:
Maybe let's just call it ::getRole? The long name leaks the implementation details.